src/main/java/org/opentripplanner/util/HttpUtils.java

```diff
@@ -52,7 +53,7 @@ public static InputStream getData(String url, String requestHeaderName, String r
     public static <T> T postData(String url, String data, Class<T> mapTo) {
         try {
             HttpPost request = new HttpPost(url);
-            request.setEntity(new StringEntity(data));
+            request.setEntity(new StringEntity(data, ContentType.APPLICATION_JSON));
```

Comment:
spoko zmiana, te `\\n` utrudniały mi życie heh

Reply:
+1

Reply:
nie działało mi mock_db więc się za to wziąłem, sory że nie w oddzielnym pr, ale potrzebowałem tego do testów więc duh